target/linux/rockchip/patches-6.6/155-add-rk806-regulator-config.patch

```diff
@@ -0,0 +1,198 @@
+From c8005a5dae74019291401f043c749e2c3fb688c8 Mon Sep 17 00:00:00 2001
+From: Lucas Tanure <lucas.tanure@collabora.com>
+Date: Mon, 9 Jan 2023 16:37:23 +0000
+Subject: [PATCH] arm64: dts: rockchip: Add RK806 regulator config
+
+Add support for rk806 single configuration PMIC in Rock Pi 5A and 5B.
+
+Signed-off-by: Lucas Tanure <lucas.tanure@collabora.com>
+Signed-off-by: Sebastian Reichel <sebastian.reichel@collabora.com>
+---
+ .../dts/rockchip/rk3588-rk806-single.dtsi     | 181 ++++++++++++++++++
+ 1 file changed, 181 insertions(+)
+ create mode 100644 arch/arm64/boot/dts/rockchip/rk3588-rk806-single.dtsi
+
+--- /dev/null
++++ b/arch/arm64/boot/dts/rockchip/rk3588-rk806-single.dtsi
```

Comment:
Upstream does not use it instead write it each dts. Should be removed.

Reply:
ok